### PR TITLE
Styling page

### PR DESCRIPTION
**Pull request by SaveryIV**
In this project:

- I followed and completed the design provided.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,325 @@
...
+  padding: 1rem 0.5rem;
+  border-radius: 2px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
+  font-family: var(--Montserrat);
+  font-weight: bolder;
+  letter-spacing: -0.15px;
+  color: var(--very-light-pink);
+}
```
Comment (brainconnect93) [inline src/App.css:80]: - [ ] Great effort working on the styling of your Bookstore project, But currently, I would like to highlight some places in your design that require your attention and do not go in line with the original styling.

1. The `book cards`, `input form`, and the `icon` should align together at the right-hand side on a straight line vertically.
2. Also your input form is missing the header test `ADD NEW BOOK` kindly work on that and be sure the text is added.
3. Also the space between one card and the other is much kindly reduce it in order to have a perfect replica of the original design.
4. Lastly, there is a straight line above the `ADD NEW BOOK` text in the original design but that is currently missing in your design.

| Original design | Your design |
| ------------- | ------------- |
| book | book1 |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @SaveryIV, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required


- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1,3 +1,337 @@
...
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
+
+header {
```
Comment (addod19) [inline src/App.css:21]: - [ ] As mentioned in the previous review, the header which contains the Bookstore text and icon should align with the BookCollection as we can see in the screenshot below
Original Design
Screenshot from 2023-09-02 06-11-21
Your Design
Screenshot from 2023-09-02 06-07-41

**Review by addod19 — CHANGES_REQUESTED** [review]
## Hi @SaveryIV, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great job fixing some of the comments :rocket: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @SaveryIV, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Aligned the header which contains the Bookstore text and icon with the BookCollection :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**